set.mm

```diff
@@ -764894,19 +764899,22 @@ According to Wikipedia ("Ackermann function", 8-May-2024,
   That the definition ~ df-ack is equivalent to P&eacute;ter's definition is
   proven by the following three theorems:
 
-  ~ ackval0val : ` ( ( Ack `` 0 ) `` M ) = ( M + 1 ) `
-  ~ ackvalsuc0val : ` ( ( Ack `` ( M + 1 ) ) `` 0 ) = ( ( Ack `` M ) `` 1 ) `
+  ~ ackval0val : ` ( ( Ack `` 0 ) `` M ) = ( M + 1 ) ` ;
+  ~ ackvalsuc0val : ` ( ( Ack `` ( M + 1 ) ) `` 0 ) = ( ( Ack `` M ) `` 1 ) ` ;
   ~ ackvalsucsucval : ` ( ( Ack `` ( M + 1 ) ) `` ( N + 1 ) ) `
-                      ` = ( ( Ack `` M ) `` ( ( Ack `` ( M + 1 ) ) `` N ) ) `.
+                      ` = ( ( Ack `` M ) `` ( ( Ack `` ( M + 1 ) ) `` N ) ) ` .
 
   The initial values of the Ackermann function are calculated in the following
   four theorems:
 
-  ~ ackval0012 : ` A ( 0 , 0 ) = 1 `,  ` A ( 0 , 1 ) = 2 `, ` A ( 0 , 2 ) = 3 `
-  ~ ackval1012 : ` A ( 1 , 0 ) = 2 `,  ` A ( 1 , 1 ) = 3 `, ` A ( 1 , 3 ) = 4 `
-  ~ ackval2012 : ` A ( 2 , 0 ) = 3 `,  ` A ( 2 , 1 ) = 5 `, ` A ( 2 , 3 ) = 7 `
-  ~ ackval3012 : ` A ( 3 , 0 ) = 5 `,  ` A ( 3 , 1 ) = ; 1 3 `,
-                 ` A ( 3 , 3 ) = ; 2 9 `
+  ~ ackval0012 : ` A ( 0 , 0 ) = 1 ` , ` A ( 0 , 1 ) = 2 ` ,
+                 ` A ( 0 , 2 ) = 3 ` ;
+  ~ ackval1012 : ` A ( 1 , 0 ) = 2 ` , ` A ( 1 , 1 ) = 3 ` ,
+                 ` A ( 1 , 3 ) = 4 ` ;
+  ~ ackval2012 : ` A ( 2 , 0 ) = 3 ` , ` A ( 2 , 1 ) = 5 ` ,
+                 ` A ( 2 , 3 ) = 7 ` ;
+  ~ ackval3012 : ` A ( 3 , 0 ) = 5 ` , ` A ( 3 , 1 ) = ; 1 3 ` ,
+                 ` A ( 3 , 3 ) = ; 2 9 ` .
```

Comment:
Why did you insert line breaks? I thik they are not helpful at this place.

Reply:
I agree it's a pity, but after I added the spaces between the backtick and the comma, the lines were 80-character wide, which is more than the 79 maximum.

Reply:
OK, knowing the reason now I can accept this.